api-gradle/src/main/kotlin/phraseapp/tasks/deprecated/DownloadTask.kt

```diff
@@ -0,0 +1,13 @@
+package phraseapp.tasks.deprecated
+
+import org.gradle.api.tasks.TaskAction
+import phraseapp.tasks.DownloadTask
+
+@Deprecated("This task is deprecated, use DownloaderTask instead")
+abstract class DownloadTask : DownloadTask() {
+    @TaskAction
+    fun downloadDeprecated() = run {
+        println("phraseappDownload is deprecated, use phraseDownload")
```

Comment:
Prefer to use `logger.warning` (or equivalent) instead of the `println` function.